modules/drivers/sqlserver/test/metabase/driver/sqlserver_test.clj

```diff
@@ -36,7 +37,10 @@
 
 (deftest connection-spec-test
   (testing "Test that additional connection string options work (#5296)"
-    (is (= {:applicationName    "Metabase <version>"
+    (is (= {:applicationName    (format
+                                 "Metabase %s [%s]"
+                                 (or (:tag config/mb-version-info) "")
+                                 config/local-process-uuid)
```

Comment:
It seems a little weird to duplicate the contents of `mb-version-and-process-identifier` itself here. I think it's fine just to make sure `(is (= {:applicationName config/mb-version-and-process-identifier ...}))`.

Reply:
Yeah it's a bit duplicative, but on the other hand, if there happened to be some bug in the implementation of `mb-version-and-process-identifier` itself, and it was used to construct the expected value in the test, then it might be a false positive.  Always a bit of a judgement call on things like this.